delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/Target.scala

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.acls
+
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+
+/**
+  * Enumeration of possible ACL targets. A target is the location where a certain ACL is anchored.
+  */
+sealed trait Target extends Product with Serializable {
+
+  /**
+    * the string representation of the target location
+    */
+  def string: String
```

Comment:
why not use regular `toString` method for this or `Show` from `cats` here?

Reply:
the regular `toString` is already used. However I also wanted to save it in a variable